about.html

```diff
@@ -4,36 +4,101 @@
+        </div>
+      </nav>
+      <nav id="menu_expand" class="about_menu">
+        <div id="menu_ham" class="abuut_menu_img"></div>
         <div id="close_menu" class="hide_menu"></div>
-        <div class="bg_img_head"></div>
-        <div class="bg_img_c"></div>
-        <div class="bg_img_x"></div>
+        <div class="menu_vertical">
+          <a href="./index.html">Home</a>
+          <a href="./about.html">About</a>
+        </div>
+      </nav>
+      <span class="subtitle_header">"Hello! welcome to the conference"</span>
+      <h1>CREATIVE ARMANDO SUMMER 2015</h1>
+      <hr class="division_logo_header">
+      <article class="header_article_mobile">
+        <h4>
+          The Global Summer springs brings together the community of experts, academics, and developer to join together and share the latest technologies that are in the market.
+        </h4>
+      </article>
+      <article class="header_article_desktop">
+        <h4>
+          The Global Summer springs brings together the community of experts, academics, and developer to join together and share the latest technologies that are in the market.
+          This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons,
+          ways to learn and apply this new tecnologies.
+          So if youre active and engaged in the worlds of web development - free software advocates, Wikipedians, Open Knowledge, galleries, workshops, web designs, inversionists,
+          stake holders, founders - we hope you'll consider joining us this year to build a stronger, more vibrant better coding practices, better desings and technologies.
+        </h4>
+      </article>
+      <div class="last_header_part">
+        <p>Please contact us per Email for any further questions about CREATIVE ARMANDO SUMMER 2015!</p>
+        <a class="link_last_header" href="mailto:businesswitharmando@gmail.com">businesswitharmando@gmail.com</a>
+      </div>
+    </header>
+    <section class="miidle_section">
+      <h2>CREATIVE ARMANDO SUMMER 2015 Logo</h2>
+      <hr class="division_logo">
+      <p class="paragraph_mobile">CREATIVE ARMANDO SUMMER 2022 logo was decided through the logo conference from 8. June to 7. July</p>
+      <p class="paragraph_desktop">The logo of CC Global SUMMER 2022 was decided through the logo competition from 8. June to 7. July.
+        After online public voting and Selection Committees judging process, the logo from Naresh Agrawal won the prize.
+        After consultation with Mr. Agrawal we optimized the prize-winning-logo for this years summit as following:
+      </p>
+      <div class="global_icon"></div>
+      <div class="global_icon_mobile"></div>
+      <hr class="division_middle">
+    </section>
+    <section class="last_section">
+      <h2>See the past CREATIVE ARMANDO SUMMER 2015 Logo</h2>
```

Comment:
- [ ] [Optional] For uniformity, it would be nice to make use of uppercase all through. Something like this:

**SEE THE PAST CREATIVE ARMANDO SUMMER 2015 LOGO**